app/views/task/index.html.erb

```diff
@@ -1,26 +1,47 @@
-<header><h1>Reading List</h1></header>
-
-<p class="mininav">
-    <%= link_to "Add a book", "/tasks/new" %>
-</p>
-
-<table>
-  <tr class="header">
-  <td>Title</td>
-  <td>Status</td>
-  <td>Unshelve it?</td>
-  </tr>
-
-  <% @displaytasks.each do |task| %>
-    <tr>
-      <td><%= link_to "#{task[:taskname]}", "/tasks/#{task[:id]}" %></td>
-      <td><%= task[:comp_status] %></td>
-      <td><%= link_to "Delete", "/tasks/#{task[:id]}", method: :delete, data: { confirm: "Are you sure?" } %></td>
+<header class = "header">
+  <div class="header_wrapper">
+    <h1>Unigoat Book Recs</h1>
+    <nav>
+      <ul>
+        <li><%= link_to "Add a Book", "/tasks/new" %></li>
+        <li><%= link_to "Unigoats", "/people" %></li>
+        <li><%= link_to "My Goodreads", "https://www.goodreads.com/review/list/4306102?shelf=read" %></li>
+      </ul>
+    </nav>
+  </div>
+</header>
+
+<div class = "content_wrapper">
```

Comment:
CSS naming conventions use the `-` between multi-word classes rather than the `_`. So here you'd want to use `content-wrapper` and `button-wrapper` instead.